src/model/test/ZoneHVACTerminalUnitVariableRefrigerantFlow_GTest.cpp

```diff
@@ -126,3 +134,160 @@ TEST_F(ModelFixture, ZoneHVACTerminalUnitVariableRefrigerantFlow_SupplementalHea
+  EXPECT_FALSE(plantLoop.addSupplyBranchForComponent(testObject));
+  EXPECT_EQ((unsigned)5, plantLoop.supplyComponents().size());
+  ASSERT_FALSE(testObject.plantLoop());
+  // EXPECT_EQ(plantLoop, testObject.plantLoop().get());
+}
+
+TEST_F(ModelFixture, ZoneHVACTerminalUnitVariableRefrigerantFlow_AddToNodeOutdoorAirSystem) {
+  Model m;
+
+  AirLoopHVAC airLoop(m);
+  Node supplyOutletNode = airLoop.supplyOutletNode();
+
+  ControllerOutdoorAir controllerOutdoorAir(m);
+  AirLoopHVACOutdoorAirSystem outdoorAirSystem(m, controllerOutdoorAir);
+  outdoorAirSystem.addToNode(supplyOutletNode);
+
+  EXPECT_EQ(3, airLoop.supplyComponents().size());
+  EXPECT_EQ(1, outdoorAirSystem.oaComponents().size());
+  EXPECT_EQ(1, outdoorAirSystem.reliefComponents().size());
+
+  if (boost::optional<Node> OANode = outdoorAirSystem.outboardOANode()) {
+    ZoneHVACTerminalUnitVariableRefrigerantFlow testObject(m);
+
+    EXPECT_TRUE(testObject.addToNode(OANode.get()));
+    EXPECT_EQ(3, airLoop.supplyComponents().size());
+    EXPECT_EQ(3, outdoorAirSystem.oaComponents().size());
+    EXPECT_TRUE(testObject.airLoopHVACOutdoorAirSystem());
+  }
+
+  if (boost::optional<Node> reliefNode = outdoorAirSystem.outboardReliefNode()) {
+    ZoneHVACTerminalUnitVariableRefrigerantFlow testObject(m);
+    EXPECT_TRUE(testObject.addToNode(reliefNode.get()));
+    EXPECT_EQ(3, airLoop.supplyComponents().size());
+    EXPECT_EQ(3, outdoorAirSystem.reliefComponents().size());
+    EXPECT_TRUE(testObject.airLoopHVACOutdoorAirSystem());
+  }
+}
+
+TEST_F(ModelFixture, ZoneHVACTerminalUnitVariableRefrigerantFlow_controllingZoneorThermostatLocation) {
+
+  // Used only for AirloopHVAC equipment on a main branch and defines zone name where thermostat is located.
+  // Not required for zone equipment. Leave blank if terminal unit is used in AirLoopHVAC:OutdoorAirSystem:EquipmentList.
+  // Required when terminal unit is used on main AirloopHVAC branch and coils are not set point controlled.
+  // When terminal unit is used in air loop and is load controlled, this zone's thermostat will control operation.
+
+  Model m;
+  ZoneHVACTerminalUnitVariableRefrigerantFlow testObject(m);
+
+  AirLoopHVAC airLoop(m);
+  Node supplyOutletNode = airLoop.supplyOutletNode();
+  EXPECT_TRUE(testObject.addToNode(supplyOutletNode));
+
+  ThermalZone z(m);
+  ScheduleConstant sch(m);
+  AirTerminalSingleDuctConstantVolumeNoReheat atu(m, sch);
+  EXPECT_TRUE(airLoop.addBranchForZone(z, atu));
+
+  EXPECT_FALSE(testObject.controllingZoneorThermostatLocation());
+  EXPECT_TRUE(testObject.setControllingZoneorThermostatLocation(z));
+  ASSERT_TRUE(testObject.controllingZoneorThermostatLocation());
+  EXPECT_EQ(z, testObject.controllingZoneorThermostatLocation().get());
+  testObject.resetControllingZoneorThermostatLocation();
+  EXPECT_FALSE(testObject.controllingZoneorThermostatLocation());
+}
+
+TEST_F(ModelFixture, ZoneHVACTerminalUnitVariableRefrigerantFlow_SupplyAirFanPlacement) {
```

Comment:
And the other new one, including checking that it's correctly defaulted in both Ctors to "DrawThrough" (got bit by it in Reg test originally)